CloudM.Exercise.WebApi/Controllers/PackageController.cs

```diff
@@ -17,6 +19,8 @@ public PackageController()
         [Route("{name}/{version}")]
         public async Task<PackageDependencies> Get(string name, string version)
```

Comment:
If we want to be more flexible with the data we return, it might be a better option to return a Task<ActionResult<PackageDependencies>>. This way we can return response codes other than 200OK, if the need arises. This may also improve error handling.